game/src/main/kotlin/world/gregs/voidps/world/interact/entity/player/combat/consume/drink/Overload.kts

```diff
@@ -11,17 +13,32 @@ import world.gregs.voidps.engine.timer.timerStart
 import world.gregs.voidps.engine.timer.timerStop
 import world.gregs.voidps.engine.timer.timerTick
 import world.gregs.voidps.world.interact.entity.combat.hit.directHit
+import world.gregs.voidps.world.interact.entity.combat.inWilderness
 import world.gregs.voidps.world.interact.entity.player.combat.consume.canConsume
 
-fun inWilderness() = false
-
 canConsume("overload*") { player ->
+    if (player.inWilderness) {
+        player.message("You cannot drink an overload potion while you're in the wilderness.", ChatType.Game)
+        cancel()
+        return@canConsume
+    }
+
     if (player.timers.contains("overload")) {
         player.message("You may only use this potion every five minutes.")
         cancel()
+        return@canConsume
     } else if (player.levels.get(Skill.Constitution) < 500) {
         player.message("You need more than 500 life points to survive the power of overload.")
         cancel()
+        return@canConsume
+    }
+}
+
+enterArea("wilderness*") {
+    if (player.timers.contains("overload")) {
+        player.timers.stop("overload")
+        player["overload_refreshes_remaining"] = 0
```

Comment:
I'm not sure if we should be removing it, or just letting it not boost. 

Aka if a player is fighting kbd, exits through the wilderness, then re-enters I imagine they'd lose their current stats, but once the timer ticks it'd get re-boosted so long as they're in kbd area?

Reply:
Yeah you're right i am just fixing that now